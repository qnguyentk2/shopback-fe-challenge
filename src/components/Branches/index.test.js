import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import '../../setupTests';

import Index from './index';

test('renders without crashing', () => {
  const wrapper = shallow(<Index />);
  expect(toJSON(wrapper)).toMatchSnapshot();
  // expect(wrapper.html()).toMatch(/Popular/);
});