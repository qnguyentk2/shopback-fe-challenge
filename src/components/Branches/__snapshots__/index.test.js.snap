// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="branches"
>
  <div
    className="wrapper"
  >
    <div
      className="branches__tabs"
    >
      <Tabs
        activeKey="0"
        hideAdd={false}
        tabPosition="top"
      />
    </div>
  </div>
</div>
`;
